scenarios/kubernetes_verify.py

```diff
@@ -52,16 +52,25 @@ def check(*cmd):
     print >>sys.stderr, 'Run:', cmd
     subprocess.check_call(cmd)
 
+def get_tag(branch):
+    """Gets the tag associated with a branch."""
+    if branch == 'master':
+        return BRANCH_VERSION.get('master')
+
+    release = re.match(r'release-(\d+\.\d+)', branch)
```

Comment:
Please copy over the comment about 3-part versions for future reference. I added it to explain the fix in #2801.